torch/csrc/jit/python_interpreter.cpp

```diff
@@ -45,9 +46,7 @@ Operation createPythonOperation(Node* op_) {
         py_inputs[i] = py::reinterpret_borrow<py::object>(
             op->scalar_args[next_scalar++].get());
```

Comment:
Do we still use those static arguments for anything? It would be better to embed them as constants in the graph

Reply:
I think they are still used for calls to python Function objects in the tracer.